Nodejs/Product/Nodejs/NpmUI/NpmWorker.cs

```diff
@@ -106,74 +106,135 @@ private void Run()
 
             TelemetryHelper.LogSearchNpm();
 
-            var relativeUri = string.Format("/-/v1/search?text=\"{0}\"", WebUtility.UrlEncode(filterText));
-            var searchUri = new Uri(defaultRegistryUri, relativeUri);
-
-            var request = WebRequest.Create(searchUri);
-            using (var response = await request.GetResponseAsync())
+            string relativeUri;
```

Comment:
There's no reason to share this across cases.

Reply:
yep